dashboard/app/components/SideBarComponents/CreateJobModal.tsx

```diff
@@ -90,62 +76,27 @@ export default function CreateJobModal({ isOpen, setOpen }: ModalProps) {
     }
   }
 
-  const inputs: Input<Job>[] = [
-    {
-      name: "name",
-      type: "text",
-      label: NAME_INPUT_LABEL,
-      required: true,
-      validationType: validateName,
-    },
-    {
-      name: "audioFile",
-      type: "file",
-      label: AUDIO_FILE_LABEL,
-      required: true,
-      inputLabelProps: {
-        shrink: true,
-      },
-      inputProps: {
-        accept: "audio/*",
-      },
-    },
-    {
-      name: "participants",
-      type: "text",
-      label: PARTICIPANTS_INPUT_LABEL,
-      required: false,
-      helperText: PARTICIPANTS_HELPER_TEXT,
-      validationType: NUMBERS,
-    },
-    {
-      name: "language",
-      type: "select",
-      label: LANGUAGE_INPUT_LABEL,
-      required: false,
-      selectOptions: LANGUAGE_DATA,
-    },
-  ];
-
   const bulkInputs: Input<BulkJob>[] = [
     {
       name: "name",
       type: "text",
       label: NAME_INPUT_LABEL,
       required: true,
       validationType: validateName,
+      useFullWidth: true,
     },
     {
       name: "audioFile",
       type: "file",
-      label: ZIP_FILE_LABEL,
+      label: isSingleInput ? AUDIO_FILE_LABEL : ZIP_FILE_LABEL,
```

Comment:
Where does this come from?

Reply:
As we want to have different labels for the file input depending on the type of file input form (single or bulk) we have to differ here. 

That line is actually a bugfix of the bulk job implementation, I noticed it while working on this issue.